pints/_log_priors.py

```diff
@@ -95,6 +97,93 @@ def sample(self, n=1):
         return np.random.beta(self._a, self._b, size=(n, 1))
 
 
+class BinomialLogPrior(pints.LogPrior):
+    r"""
+    Defines a binomial (log) prior with given number of trials parameter
+    ``trials`` and trial success probability parameter ``prob`` with
```

Comment:
It's a good rule of thumb to start each docstring with a 1-line paragraph without latex (some IDEs show this when users type the name of your class/method).
So maybe stick a period after ``prop``, then a newline, then "The PDF is given by"